android/sdl_android/src/androidTest/java/com/smartdevicelink/test/VehicleDataHelper.java

```diff
@@ -93,6 +93,13 @@
     public static final String OEM_CUSTOM_VEHICLE_DATA_STATE = "oemCustomVehicleDataState";
     public static final Boolean HANDS_OFF_STEERING = Boolean.TRUE;
 
+    // Seat Occupancy
+    public static final SeatOccupancy SEAT_OCCUPANCY = new SeatOccupancy();
+    public static final List<SeatStatus> SEATS_OCCUPIED = new ArrayList<SeatStatus>(1);
```

Comment:
@kostyaBoss Line 97 is in the right place, but you can move the other variables on lines 98-101 back to where they were before.  

Line 97 `SEAT_OCCUPANCY` is the top level variable while the other variables on lines 98-101 are lower variables.